jenkins/scripts/select-compiler.sh

```diff
@@ -4,39 +4,41 @@ if [ "$DONTSELECT_COMPILER" != "DONT" ]; then
   NODE_NAME=${NODE_NAME:-$HOSTNAME}
   echo "Selecting compiler based on $NODE_NAME"
   case $NODE_NAME in
-    *ppc64_le* ) SELECT_ARCH=PPC64LE ;;
+    *ppc64*le* ) SELECT_ARCH=PPC64LE ;;
     *s390x* ) SELECT_ARCH=S390X ;;
     *aix* ) SELECT_ARCH=AIXPPC ;;
   esac
 fi
 
 # get node version
-if [ -z ${NODEJS_MAJOR_VERSION+x} ]; then
+if [ -z "$NODEJS_MAJOR_VERSION" ]; then
   NODE_VERSION="$(python tools/getnodeversion.py)"
   NODEJS_MAJOR_VERSION="$(echo "$NODE_VERSION" | cut -d . -f 1)"
 fi
 
 if [ "$SELECT_ARCH" = "PPC64LE" ]; then
-  # Set default
-  export COMPILER_LEVEL="4.8"
-
   echo "Setting compiler for Node version $NODEJS_MAJOR_VERSION on ppc64le"
 
   if [ "$NODEJS_MAJOR_VERSION" -gt "11" ]; then
-    # See: https://github.com/nodejs/build/pull/1723#discussion_r265740122
-    export PATH=/usr/lib/binutils-2.26/bin/:$PATH
-    export COMPILER_LEVEL="6"
-  elif [ "$NODEJS_MAJOR_VERSION" -gt "9" ]; then
-    export PATH=/usr/lib/binutils-2.26/bin/:$PATH
-    export COMPILER_LEVEL="4.9"
-  fi
+    # Setup devtoolset-6, sets LD_LIBRARY_PATH, PATH, etc.
```

Comment:
We need to support both ubuntu and centos so this will need to check that and then ensure the right compiler is selected based on the os